six/projects/csm/source/SICDSensorModel.cpp

```diff
@@ -387,6 +383,12 @@ csm::ImageVector SICDSensorModel::getImageSize() const
     return csm::ImageVector(mData->getNumRows(), mData->getNumCols());
 }
 
+csm::ImageCoord SICDSensorModel::getImageStart() const
+{
+    return csm::ImageCoord(mData->imageData->firstRow,
```

Comment:
Add a comment here and in the SIDD one summarizing what John explained as the convention in #279.  That way when we look back at this in two years we'll be able to convince ourselves we're accounting for AOIs the right way.